erica/elster_xml/common/basic_xml_data_representation.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ENutzdaten:
```

Comment:
It is for Elster, yes. You think I should rename it to "ElsterNutzdaten" etc? Or just mention it in the Docstring?

Reply:
Personally, I would prefer ElsterNutzdaten then

Reply:
Just my 2cents: even though it may not be immediately obvious what the E prefix stands for, once you learn its meaning everything will makes sense without any additional effort. Wouldn't it be nicer to keep the E and learn its meaning once rather than changing it to Elster in every class name and thus having to deal with consistently longer (and similar looking and hence harder to read) class names?

Reply:
> in every class name and thus having to deal with consistently longer (and similar looking and hence harder to read) class names?

That's right. What about putting Elster at the end. The problem I see is just that E is used in the Elster context quite often for different things (EDaten, the ENumber). Therefore the initial assumption might lead you to the wrong conclusion (I assumed this might hold the ENumber representation).

Reply:
Actually, if I'm really honest, I very much prefer the E-prefix. I definitely see your point with the potential of confusion, but this doesn't really offset the disadvantage of very ugly names for me. I've just grown quite fond of the E-prefix. So if you don't veto, I'd like to leave it as is.

Reply:
I won’t veto:)